File: frontend/app/src/main/java/com/example/commov/ComMovApplication.java
package com.example.commov;

import android.app.Application;

import com.example.commov.data.local.LocaleHelper;

public class ComMovApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        LocaleHelper.applySavedAppLocale(this);
    }
}

File: frontend/app/src/main/java/com/example/commov/ComMovApplication.kt
package com.example.commov

import android.app.Application
import com.example.commov.data.local.LocaleHelper
import com.example.commov.data.sync.ProfilePhotoNotifier
import com.example.commov.data.sync.ProfilePhotoSyncManager

class ComMovApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        LocaleHelper.applySavedAppLocale(this)
        ProfilePhotoNotifier.ensureChannel(this)
        Thread {
            ProfilePhotoSyncManager.syncIfNeeded(this)
        }.start()
    }
}
